src/NexusMods.App.UI/Controls/GameWidget/GameWidget.axaml.cs

```diff
@@ -0,0 +1,63 @@
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using Avalonia.Media.Imaging;
+using Avalonia.ReactiveUI;
+using ReactiveUI;
+using SkiaSharp;
+
+namespace NexusMods.App.UI.Controls.GameWidget;
+
+public partial class GameWidget : ReactiveUserControl<IGameWidgetViewModel>
+{
+    public GameWidget()
+    {
+        InitializeComponent();
+        this.WhenActivated(d =>
+        {
+            this.Bind(ViewModel, vm => vm.Image, v => v.GameImage.Source)
+                .DisposeWith(d);
+            this.WhenAnyValue(view => view.ViewModel!.Image)
+                .Where(img => img != null)
```

Comment:
Regarding nullability with RxUI: https://github.com/reactiveui/ReactiveUI/issues/2469 describes the problem very well, since `WhenAnyValue` accepts an `Expression`, we can't use the `?` [null-conditional operator](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/member-access-operators#null-conditional-operators--and-). This issue is also being discussed in https://github.com/dotnet/csharplang/discussions/158.

I think our best option going forward is to use `!` for now, until support for `?` is implemented, and use the `WhereNotNull` extension of RxUI to prevent any null exceptions:

```csharp
this.WhenAnyValue(view => view.ViewModel!.Image)
  .WhenNotNull()
```

The `WhenNotNull` extension has a simple not null check and converts nullability:

```csharp
    public static IObservable<T> WhereNotNull<T>(this IObservable<T?> observable) =>
        observable
            .Where(x => x is not null)
            .Select(x => x!);
```

If C# adds support for `?` inside expressions, we won't have to change much code since `WhereNotNull` already converts nullability.